test/client/spec/victory-core/victory-primitives/point.spec.js

```diff
@@ -36,9 +35,7 @@ describe("victory-primitives/point", () => {
       "star",
       "cross"
     ].forEach((symbol) => {
-      const stub = sandbox
-        .stub(pathHelpers, symbol)
-        .returns(`${symbol} symbol`);
+      stub = sinon.stub(pathHelpers, symbol).returns(`${symbol} symbol`);
```

Comment:
If there is an error in the test _before_ this line of code your `stub.restore()` call in line 23 with all fail. Better to switch back to the sandbox approach because that is guaranteed to be instantiated in `beforeEach`